Makefile

```diff
@@ -843,10 +843,10 @@ $(O_FILES): | asset_files
 
 .PHONY: o_files asset_files
 
-$(BUILD_DIR)/$(SPEC): $(SPEC) $(SPEC_INCLUDES)
-	$(CPP) $(CPPFLAGS) $< | $(BUILD_DIR_REPLACE) > $@
+$(BUILD_DIR)/spec: $(SPEC) $(SPEC_INCLUDES)
```

Comment:
Perhaps I'm missing something, but does `SPEC_INCLUDES` actually do anything here? Whatever is used is just found from a wildcard, so would mean it exists and does not have to be a requirement here?

Reply:
Ahh, that makes sense. Thanks for the clarification.